tests/unit/mock-api-data/text_androids.json

```diff
@@ -0,0 +1,1149 @@
+[
+  {
+    "text": "do androids dream of electric sheep",
+    "score": null,
+    "pos_types": null,
+    "fingerprint": {
+      "positions": [
```

Comment:
This seems wrong. It's this endpoint, right? http://api.cortical.io/Text.htm#!/text/getRepresentationForText_post_0

When I make this request: POST http://api.cortical.io/rest/text

Headers:

    {"Content-Type": "application/json", "api-key": "43b3bbf0-f243-11e3-80ac-f7122a45615d", "Accept": "Application/json"}

Query params:

    {"retina_name": "en_synonymous"}

Post data:

    do androids dream of electric sheep

I get back a JSON structure like this:

    [{"positions":[...]}]